frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/children/index.tsx

```diff
@@ -107,35 +101,38 @@ export default function ApplyFlowChildSummary() {
       <div className="max-w-prose">
         <fetcher.Form method="post" aria-describedby="form-instructions-sin form-instructions" noValidate>
           <input type="hidden" name="_csrf" value={csrfToken} />
-          {!hasChildren && <p>{t('apply-adult-child:children.index.no-children')}</p>}
-          {hasChildren && (
+          {children === undefined || children.length == 0 ? (
```

Comment:
```suggestion
          {children?.length === 0 ? (
```

Reply:
Amended to use `hasChildren` var instead. BTW, `children?.length === 0` wouldn't work if `children` is `undefined because `undefined` doesn't equals to `0`.

Reply:
> `undefined` doesn't equals to `0`

My bad. You are right.